src/main/java/de/dafuqs/spectrum/registries/client/SpectrumModelPredicateProviders.java

```diff
@@ -247,12 +248,7 @@ private static void registerSlotReservingItem(Item item) {
 	}
 	
 	private static void registerOversizedItemPredicate(Item item) {
-		ModelPredicateProviderRegistry.register(item, new Identifier("in_world"), (itemStack, world, livingEntity, i) -> {
-			if (world == null && livingEntity == null && i == 0) { // REIs 'fast batch' render mode. Without mixin' into REI there is no better way to catch this, I am afraid
-				return 0.0F;
-			}
-			return currentItemRenderMode == ModelTransformationMode.GUI || currentItemRenderMode == ModelTransformationMode.GROUND || currentItemRenderMode == ModelTransformationMode.FIXED ? 0.0F : 1.0F;
-		});
+		ModelPredicateProviderRegistry.register(item, new Identifier("oversized"), (itemStack, world, livingEntity, seed) -> seed == 817210941 ? 1.0F : 0.0F);
```

Comment:
'oversized' does not really reflect what the predicate does.
It does not specify that the item should be rendered oversized at all, it specifies if a model is rendered in a gui or in-world.

Reply:
![image](https://github.com/user-attachments/assets/0f7e6170-fbab-48f5-8c1f-374132def908)